tests/test_playbooks.py

```diff
@@ -21,3 +20,24 @@ def playbook(request):
 def test_takes_package_argument(playbook):
     expected = playbook.name not in ('setup', 'cleanup-copr')
     assert playbook.takes_package_parameter == expected
+
+
+def test_is_documented(playbook):
+    assert playbook.help_text is not None
+
+
+def test_help(playbook, capsys, help_dir):
+    with pytest.raises(SystemExit) as excinfo:
+        obal.main([playbook.name, '--help'])
+    assert excinfo.value.code == 0
+
+    captured = capsys.readouterr()
+
+    help_file = help_dir / '{}.txt'.format(playbook.name)
+
+    try:
+        assert help_file.read() == captured.out
+    except IOError:
```

Comment:
I think this warrants a developer doc, how to record new help files? (I am reading this as: if the file does not exist, it will be written)